### PR TITLE
Created HUD/UI

### PR DESCRIPTION
**Pull request by wimpula0**

### CODE REVIEW

File: scenes/levels/level 1.tscn [hunk 1]
```diff
@@ -213,3 +219,85 @@ autoplay = true
 
 [node name="KeyManager" type="Node" parent="Managers"]
 script = ExtResource("6_k22wn")
+
+[node name="HUD" type="CanvasLayer" parent="."]
```
Comment (vyachkoslav) [inline scenes/levels/level 1.tscn]: Better move the HUD into the player's scene, so there's no need to add separate HUD to each level. Also use the HUD.tscn for that.

File: scripts/PlayerAnimations.gd
Comment (vyachkoslav) [inline scripts/PlayerAnimations.gd:1]: PlayerAnimations script is meant for animations by definition and we also need interface for damage receiving, so you should either move the animations script to a separate node (like PlayerMovement script) or redefine the purpose of this script.

**Review by vyachkoslav — CHANGES_REQUESTED** [review]
Move HUD to the player's scene and split animations logic with damage logic to separate scripts.

**Comment by wimpula0** [comment]
Couldn't add HUD.tscn to players scene though because of issues with the Hearts Array.

**Pull request merged**